pkg/fleet/internal/fixtures/fs.go

```diff
@@ -60,6 +61,11 @@ func fsContainsAll(a fs.FS, b fs.FS) error {
 		if err != nil {
 			return err
 		}
+		// On Windows we have pesky \r that will mess our test result.
```

Comment:
are the `\r` there because of the git setting `core.autocrlf=true` ?

if that's the case I think I'd rather put a `.gitattributes` file in the fixtures dir to tell git to keep the LF, rather than add code to the test to work around git configuration/behavior.

content would look something like
https://github.com/DataDog/datadog-agent/blob/main/.gitattributes#L1-L2

it looks like the fixtures only contain text files right now, but the `bytes.ReplaceAll` would corrupt binary files, too, which we don't want. idk if this assert func is used elsewhere or not.